website/docs/components/switch/switch-a11y.md

```diff
@@ -11,47 +11,38 @@ tabs: Design('switch'), A11y('switch-a11y'), API('switch-api'), Example('switch-
 
 Table: Keyboard support
 
-| Key              | Function                                               |
-| ---------------- | ------------------------------------------------------ |
+| Key               | Function                                               |
+| ----------------- | ------------------------------------------------------ |
 | `Space` , `Enter` | Changes state of the switch to checked or not checked. |
 
 ### Roles & attributes
 
 The list below describes roles and attributes that component already has.
 
-| Role   | Attribute              | Element  | Usage                                                                                                                                                                          |
-| ------ | ---------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| switch |                        | `button` | When you can, use a `<button>` element and include `aria-pressed` or `role="switch"` for your toggles.                                                                         |
-|        | `aria-checked="true"` | `button` | Indicates the switch is on. CSS attribute selectors (for example [ `aria-checked="true"` ]) are used to synchronize the visual states with the value of the `aria-checked` attribute.   |
-|        | `aria-checked="false"` | `button` | Indicates the switch is off. CSS attribute selectors (for example [ `aria-checked="false"` ]) are used to synchronize the visual states with the value of the `aria-checked` attribute. |
-|        | `aria-readonly` | `button` | Indicates that the element isn’t editable, but is otherwise operable.                                                                                                         |
+| Component      | Role     | Attribute              | Usage                                                                                                                                                                                   |
+| -------------- | -------- | ---------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `Switch.Value` | `switch` | `aria-checked="true"`  | Indicates the switch is on. |
+| `Switch.Value` | `switch` | `aria-checked="false"` | Indicates the switch is off. |
```

Comment:
Maybe merge these 2 true/false rows into one? You can leave just "aria-checked" in the attribute column, and clarify that it can be true or false and indicates whether the switch is on or off.

Reply:
Yeah, agree!